doc/source/release-notes/Percona-XtraDB-Cluster-5.6.46-28.38.rst

```diff
@@ -0,0 +1,30 @@
+.. rn:: 5.6.46-28.38
+
+================================================================================
+|PXC| |release|
+================================================================================
+
+Percona is glad to announce the release of |PXC| |release|
+on |date|.  Binaries are available from the `downloads section
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/>`_ or from our
+:ref:`software repositories <installation>`.
+
+|PXC| |release| is now the current release, based on the following:
+
+- `Percona Server 5.6.46
+  <https://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.46-86.2.html>`_
+- Codership WSREP API release 5.6.46
+- Codership Galera library 3.28
```

Comment:
Galera library version is 3.38
```
mysql> show status like '%wsrep_provider_version%';
+------------------------+----------------+
| Variable_name          | Value          |
+------------------------+----------------+
| wsrep_provider_version | 3.38(rb3295e6) |
+------------------------+----------------+
1 row in set (0.00 sec)

mysql>

Reply:
3.28 is the correct Codership version.  PXC versions are different.